src/styles.css

```diff
@@ -0,0 +1,44 @@
+* {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", system-ui,
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Web", sans-serif;
+  text-align: center;
+  box-sizing: border-box;
+  -webkit-font-smoothing: antialiased;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", system-ui,
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Web", sans-serif;
+  margin: 0;
+}
+
+.teams-container {
+  width: 100vw;
+  height: 100vh;
+  margin: 0px;
+  padding: 0px;
+}
+
+.teams-header {
+  display: flex;
+  height: 6%;
+  width: 100%;
+}
+
+.teams-content {
+  width: 100%;
+  height: 94%;
+  display: flex;
+}
+
+.teams-sidebar {
+  display: flex;
+  width: 4%;
+  height: 100%;
+}
+
+.teams-body {
+  display: flex;
+  width: 96%;
+  height: 100%;
+}
```

Comment:
1. Why do we need `display: flex` in case of `teams-header, sidebar and body`?
2. Fixed width to `Header` and `Sidebar` is fine but we should use flex grow for `Content` and `Body`.